.travis.yml

```diff
@@ -17,6 +17,9 @@ install:
     # Keep track of Pyro dev branch
     - pip install https://github.com/pyro-ppl/pyro/archive/dev.zip
 
+    # Keep track of pyro-api master branch
+    - pip install https://github.com/pyro-ppl/pyro-api/archive/master.zip
```

Comment:
@neerajprad I've added this so we can use numeric versioning in setup.py, which is required by PyPI